device-types/Arctic-Wolf/awn1000-10g.yaml

```diff
@@ -0,0 +1,89 @@
+---
+manufacturer: Arctic Wolf
+model: AWN1000 10G
+slug: arctic-wolf-awn1000-10g
+description: 10G security sensor
+part_number: AWN1000 10G
+u_height: 1.0
+is_full_depth: true
+comments: '[Product documentation](https://docs.arcticwolf.com/bundle/sensors/page/deploy_an_awn1000_10g_sensor_with_internal_tap.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS-1
+    type: iec-60320-c14
+  - name: PS-2
+    type: iec-60320-c14
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data can you model the removable PSUs under `module-bays` instead of `power-ports`.

If you have any questions, please let us know :)

Reply:
I have replaced the `power-ports` with `module-bays`, though I'm not totally sure the PSUs are meant to be removed by an end-user.